installation/environment-variables.md

```diff
@@ -57,18 +57,65 @@ description: >-
   </tbody>
 </table>
 
-## Examples - Linux
+## Examples - Kubernetes \(yml configuration\)
```

Comment:
do we need to add "\" prefix for parantheses? I also saw bunch of them on processors page and removed but not sure.

Reply:
It was already there, removed it.